src/schematools/validation.py

```diff
@@ -119,9 +119,9 @@ def run_all(self) -> Iterator[ValidationError]:
         """Run all registered validators.
 
         Yields:
-            :class:`ValidationError`s if any.
+            :class:`ValidationError`\s if any.
 
-        """
+        """  # noqa W605
```

Comment:
This is incorrect syntax for `noqa` (and really common problem). This will hide ALL PEP8 errors:

Error code should be added after semicolon: `# noqa: W605`

---
Example:

```
x = """
     really long ass line to show that we have 2 errors in same block for real, no joke, best line ever
     :class:`ValidationError`\s if any.
x"""
```

Will result in:
```
test.py:4:80: E501 line too long (103 > 79 characters)
test.py:5:30: W605 invalid escape sequence '\s'
```

and with `# noqa: W605`

``` 
flake8 test.py       
test.py:4:80: E501 line too long (103 > 79 characters)
```

while `# noqa W605` will say file is completely correct.

Reply:
Excellent catch. Thank you. Will fix